source/documentors/references/doc_style_guide.rst

```diff
@@ -0,0 +1,412 @@
+
+Also, you can do a :glob: `*` on an index page, meaning in some cases, documents in a page tree are automatically added to the ToC. In the following example, any file contained in the same folder will automatically be added to the table of contents. It is a quick way to set up a table of contents so that new documents are automatically picked up, but it reduces the control you have on the order of your documents in the TOC:
+
+.. code-block::
+
+    .. toctree::
+       :maxdepth: 1
+       :glob:
+
+       *
+
+Tables
+======
+
+When you include a table, be sure to include a heading row. In addition, consider whether a stub column is appropriate. The heading row and stub column provide useful context for users of screen readers.
+
+Tables should be used to compile complicated data and indicate its relationships. In Open edX documentation, tables look like this:
+
+.. list-table:: Title
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Heading row 1, column 1
+     - Heading row 1, column 2
+     - Heading row 1, column 3
+   * - Row 1, column 1
+     -
+     - Row 1, column 3
+   * - Row 2, column 1
+     - Row 2, column 2
+     - Row 2, column 3
+
+• When the information is significantly complicated.
+• When the information repeats at least one element more than twice.
+• When the information does not benefit from being compressed.
+
+When not to use a table:
+• If that information could be better written in a sentence.
+• The information is important but not sufficiently complicated.
+
+For information on entering tables in RST, see the RST guide.
+
+Tags
+====
+
+Tags appear as links (e.g. #Administrator) at the top of the article. If you click on the tag, it will take you to a page listing all files with that tag. This allows users to search for documents with a particular tag. A typical document should have at least two tags:
+- One or more persona tags: The document is tagged with the intended reader persona type. If the document is relevant to more than one audience, then you may use more than one tag. Educator, developer, site operator, community, documentor, translator.
+- One (and only one) diataxis tag: The document is tagged with the diataxis type it belongs to. Concept, reference, quick-start, how-to.
+
+The syntax of tags is as follows:
+
+.. tags:: educator, reference
+
+NOTE: Tags should be placed after the first heading in the document.
+
+Use of Abbreviations
+====================
+
+The first time you want to use an abbreviation, spell it out. For example:
+
+The Open edX `docs.openedx.org` repository is using a documentation markup language called reStructuredText (RST) which is very similar to Markdown. The following guide is helpful to refer to as you produce RST documentation: RST Guide.
+
+Punctuation
+===========
+
+Punctuation is the primary example of a convention. For specifics on punctuation use, see https://draft-edx-style-guide.readthedocs.io/en/latest/global_English.html#.
```

Comment:
This link is wrong, we are getting rid of this site. Should this refer to another guide in docs.openedx.org?

Reply:
Yes, I've just replaced it.

Reply:
This is still the wrong link, we should not link to the draft-edx-style-guide

Reply:
@jesgreco can you address this comment?

Reply:
Done :)